### ISSUE TITLE
RFE: allow text wrapping

### ISSUE THREAD

**Issue by nono-gdv** [sni/Thruk#1224]
Hello,

The Service Status Details view truncates the Status information column (in my case, it is the rightmost displayed column) to fit the table into the browser width. Wrapping the text would make it much easier to read than having to grab the mouse and get the popup to show (not to mention it would enable copy-paste).

![screenshot](https://files.glou.org/thruk2.png)

**Comment by sni** [comment]
i see, but wrapping the text would totally break the layout of the page. Right now, each row has a known and defined height. Wrapping the text would make the table dynamic and uncontrollable.

**Comment by dan-m-joh** [comment]
I agree with Sven, dynamic resizing would ,in my opinion, make the table "messy".

Regarding Copy/Paste: I can "triple click" on a "Status information" row and then the whole row gets selected, inclusive not visible content.

For example:
![image](https://user-images.githubusercontent.com/26163222/211284952-4665ebb1-9521-48bf-9108-8d11c0514ba2.png)
Triple click on the "QUERY ..."
![image](https://user-images.githubusercontent.com/26163222/211285248-2dc991ac-b7c1-4cff-a634-8b7614b04c10.png)
Now you can "right click" on the selected text and select "Copy"
![image](https://user-images.githubusercontent.com/26163222/211285602-f919f212-3b93-48e1-aa78-33723b691e0f.png)
And voila, you have the whole text in your "copy buffer":
`QUERY OK: 'SELECT TIMESTAMPDIFF(MINUTE, FROM_UNIXTIME(value), NOW()) FROM eccf7_status WHERE name="last_update"' returned 1.000000`
Regards
Dan
